pytest_motor/plugin.py

```diff
@@ -1,19 +1,19 @@
 """A pytest plugin which helps test applications using Motor."""
 import asyncio
-import platform
 import secrets
 import shutil
 import socket
-import tarfile
-import tempfile
 from pathlib import Path
-from typing import AsyncIterator, Iterator, List, Tuple
+from typing import AsyncIterator, Iterator, List
 
-import aiohttp
 import pytest
 from _pytest.config import Config as PytestConfig
 from motor.motor_asyncio import AsyncIOMotorClient
 
+from .mongod_binary import MongodBinary
+
+# pylint: disable=redefined-outer-name
```

Comment:
Do you think that pulling this disable up here will cause us to miss redefinitions that otherwise would have been caught?

Reply:
The reason I did that is it wasn't enabling anywhere.

So it probably needs to be like this:

```python
# pylint: disable=redefined-outer-name
def test_function_with_redefinition(redefinition):
    # pylint: enable=redefined-outer-name
    pass
```

Reply:
Good point. Inlining when we can would be even better:

```python3
def test_function_with_redefinition(redefinition):  # pylint: disable=redefined-outer-name
    pass
```

Reply:
Pylint seems to work differently. It ignores errors in the current scope, not line-by-line.

This does the trick:

```python
def test_function_with_redefinition(redefinition):
    # pylint: disable=redefined-outer-name
    pass
```